JoyboyCommunity/src/assets/icons.tsx

```diff
@@ -107,6 +107,90 @@ export const UserIcon: React.FC<SvgProps> = (props) => (
   </Svg>
 );
 
+export const ExitIcon: React.FC<SvgProps> = (props) => (
+  <Svg width={57} height={56} fill="none" {...props}>
+    <Rect width={56} height={56} x={0.5} fill="currentColor" fillOpacity={0.1} rx={28} />
+    <Path
+      fill="currentColor"
+      fillRule="evenodd"
+      d="M29.35 20c-4.811 0-8.6 3.642-8.6 8 0 4.358 3.789 8 8.6 8 .43 0 .852-.03 1.264-.086a1 1 0 0 1 .272 1.982c-.502.069-1.015.104-1.536.104-5.792 0-10.6-4.417-10.6-10s4.808-10 10.6-10c.521 0 1.034.035 1.536.104a1 1 0 1 1-.272 1.982A9.305 9.305 0 0 0 29.35 20Z"
+      clipRule="evenodd"
+    />
+    <Path
+      fill="#EC796B"
+      d="M27.25 29.006a1 1 0 0 1 0-2h6.5v-.594c0-.176 0-.392.022-.568v-.004c.016-.126.088-.702.653-.976.567-.275 1.067.027 1.176.092l.468.34c.376.294.89.7 1.281 1.08.195.19.397.407.556.638.14.205.344.555.344.986 0 .43-.203.78-.344.986-.16.231-.36.448-.556.638-.39.38-.905.786-1.28 1.08l-.47.34c-.108.065-.608.367-1.175.092-.565-.274-.637-.85-.653-.976v-.004c-.022-.176-.022-.392-.022-.568v-.582h-6.5Z"
+    />
+  </Svg>
+);
+
+export const LineIcon: React.FC<SvgProps> = (props) => (
+  <Svg width={37} height={5} fill="none" {...props}>
+    <Rect width={36} height={5} x={0.5} fill="currentColor" fillOpacity={0.3} rx={2.5} />
+  </Svg>
+);
+
+export const ProfileIcon: React.FC<SvgProps> = (props) => (
+  <Svg width={32} height={32} fill="none" {...props}>
+    <Path
+      stroke="#1E2F3D"
```

Comment:
As I've stated before, colors should be used on usage, not definition. `stroke="currentColor"` `<ProfileIcon color="#1E2F3D" />`